test.js

```diff
@@ -169,3 +170,118 @@ test('PProgress.all() with concurrency = 1', async t => {
 		end: 250 // Reasonable padding
 	}));
 });
+
+test('PProgress.allSettled()', async t => {
+	const fixtureFn = input => pProgress(async progress => {
+		progress(0.16);
+		await delay(50);
+		progress(0.55);
+		await delay(100);
+		return input;
+	});
+
+	const fixtureFn2 = input => pProgress(async progress => {
+		progress(0.14);
+		await delay(52);
+		progress(0.37);
+		await delay(104);
+		progress(0.41);
+		await delay(26);
+		progress(0.93);
+		await delay(55);
+		throw input;
+	});
+
+	const p = PProgress.allSettled([
+		delay(103),
+		fixtureFn(fixture),
+		delay(55),
+		fixtureFn2(errorFixture),
+		delay(14),
+		delay(209)
+	]);
+
+	p.onProgress(progress => {
+		t.true(progress >= 0 && progress <= 1);
```

Comment:
```suggestion
		t.true(progress >= 0 && progress <= 1, '${progress}');
```

To make the assertion more debuggable if it fails.